releasenotes/notes/document-splitter-accept-split-threshold-467abb9fcd1c316b.yaml

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
To be removed from the PR.

Reply:
But I removed it, no? See file [diffs](https://github.com/deepset-ai/haystack/pull/8042/files)